<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Danish_Khan" failures="5" tests="5" name="LoginTestTest(failed)" time="40.41" errors="0" timestamp="2023-07-19T23:23:07 IST">
  <testcase classname="com.eBanking.testCases.TC_DDT_002" name="loginDDT" time="7.852">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBanking.testCases.TC_DDT_002.loginDDT(TC_DDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.eBanking.testCases.TC_DDT_002" name="loginDDT" time="6.785">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBanking.testCases.TC_DDT_002.loginDDT(TC_DDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.eBanking.testCases.TC_DDT_002" name="loginDDT" time="6.876">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBanking.testCases.TC_DDT_002.loginDDT(TC_DDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.eBanking.testCases.TC_DDT_002" name="loginDDT" time="6.848">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBanking.testCases.TC_DDT_002.loginDDT(TC_DDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.eBanking.testCases.TC_DDT_002" name="loginDDT" time="6.75">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBanking.testCases.TC_DDT_002.loginDDT(TC_DDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- LoginTestTest(failed) -->
